### PR TITLE
Feature/implement xml file for ceneo

### PR DESCRIPTION
**Pull request by memorisanka**
Hi everyone!

I added some small changes to Ceneo integration feature:
- new field in Product model (availability) - required for Ceneo
- script for changing availability to 99 ("Check availability in store" or something like that)
- some small changes in Ceneo XML file

I'm waiting for your comments, because it's hard to write something after all this time :P

### CODE REVIEW

File: Dshop/apps/products_catalogue/views.py [hunk 1]
```diff
@@ -64,22 +64,34 @@ def generate_xml_file_for_ceneo(products):
...
 
             cat_element = etree.SubElement(o_element, 'cat')
             cat_element.text = etree.CDATA(product.category.name)
 
             name_element = etree.SubElement(o_element, 'name')
             name_element.text = etree.CDATA(product.name)
 
+            if product.featured_photos.exists():
+                featured_image_url = product.featured_photos.first().image.url
+                print(featured_image_url)
```
Comment (jacoor) [inline Dshop/apps/products_catalogue/views.py]: Please remove print

**Review by jacoor — CHANGES_REQUESTED** [review]
This looks good. Small changes.

**Comment by jacoor** [comment]
@memorisanka please update from dev too.

File: Dshop/apps/products_catalogue/views.py (hunk 1 above)
Comment (memorisanka) [reply Dshop/apps/products_catalogue/views.py]: Done, sorry for this. 
But I have a problem with migrations now: ModuleNotFoundError: No module named 'dj_shop_cart'
Tests didn't pass.

Comment (memorisanka) [reply Dshop/apps/products_catalogue/views.py]: Solved. I had to install it manually.

Comment (jacoor) [reply Dshop/apps/products_catalogue/views.py]: What do you mean manually?

**Review by jacoor — APPROVED** [review]

File: Dshop/apps/products_catalogue/views.py (hunk 1 above)
Comment (memorisanka) [reply Dshop/apps/products_catalogue/views.py]: Pip install dj... cart and pip install charset-normalizer. I had errors using poetry update.

**Pull request merged**